test/unit/base_client.test.ts

```diff
@@ -168,6 +168,26 @@ describe('Base Client', () => {
         externalSystem: 'test',
       });
     });
+    test('invoke method with eventId and SkillId', async () => {
```

Comment:
I would add also a negative test to see what is returning when the skill id is wrong or does not exists